pandas/tools/plotting.py

```diff
@@ -1134,6 +1134,31 @@ def _compute_plot_data(self):
 
         self.data = numeric_data
 
+    def _compute_plot_data_with_dt(self):
```

Comment:
This causes other plots (bar, line...) to include datetime columns as values. It should be specially handled for ``x`` and ``y`` (datetime columns are not valid for scatter's ``s`` or ``c`` arguments).